py/bitbox02/bitbox02/bitbox02/bitbox02.py

```diff
@@ -1268,3 +1268,21 @@ def bluetooth_upgrade(self, firmware: bytes) -> None:
                 break
             else:
                 raise Exception(f"Unexpected response: f{response_type}")
+
+    def bluetooth_enable(self, enable: bool) -> None:
+        """
+        Enable/disable blueooth in non-volatile storage
+        """
+
+        request = bluetooth.BluetoothRequest()
+        request.enable.CopyFrom(bluetooth.BluetoothEnableRequest(enable=enable))
+
+        response = self._bluetooth_msg_query(request)
```

Comment:
`...request, expected_response="success")` (if you end up changing to `success`), so you don't need to manually check the response type below

Reply:
done